gce/image/build_image.sh

```diff
@@ -195,6 +195,7 @@ echo "Calling Packer..."
   -var scylla_tools_version="$SCYLLA_TOOLS_VERSION" \
   -var scylla_python3_version="$SCYLLA_PYTHON3_VERSION" \
   -var scylla_build_id="$BUILD_ID" \
+  -var python="/usr/bin/python3" \
```

Comment:
I think you can drop this change, we should remove this script as we build only for ubuntu based